app/config/projects.yml

```diff
@@ -286,10 +286,22 @@ parameters:
             keywords: [php, annotations, docblock, comment]
             versions:
                 -
-                    name: 'master'
+                    name: '2.0'
                     branchName: 'master'
                     slug: 'latest'
+                    upcoming: true
+                -
+                    name: '1.7'
+                    branchName: '1.7'
+                    slug: '1.7'
+                    upcoming: true
+                -
+                    name: '1.6'
+                    branchName: '1.6'
```

Comment:
Branch named 1.6 was never created for annotations.

Reply:
Ah, there is 1.5 and 1.7. 😂 Created from v1.6.0 tag.